doc/classes/Node.xml

```diff
@@ -830,10 +830,12 @@
 			Notification received when the node is instantiated.
 		</constant>
 		<constant name="NOTIFICATION_DRAG_BEGIN" value="21">
-			Notification received when a drag begins.
+			Notification received when a drag operation begins. All nodes receive this notification, not only the dragged one.
+			Can be triggered either by dragging a [Control] that provides drag data (see [method Control._get_drag_data]) or using [method Control.force_drag].
 		</constant>
 		<constant name="NOTIFICATION_DRAG_END" value="22">
-			Notification received when a drag ends.
+			Notification received when a drag operation ends.
+			Use [method Viewport.gui_is_drag_successful] to check if the drag succeeded and [method Viewport.gui_get_drag_data] to get the dragged data.
```

Comment:
Drag data is cleared before sending `NOTIFICATION_DRAG_END`, so calling `Viewport.gui_get_drag_data()` always return `null` :(